cmd/endorse.go

```diff
@@ -50,6 +56,8 @@ func (f *endorseCommand) AddFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().BoolVar(&f.AddTdx, "add_tdx", false,
 		"Add Intel TDX information to endorsement")
 	addUefiFlag(cmd, &f.UefiPath)
+	cmd.PersistentFlags().StringVar(&f.SvsmPath, "svsm_path", "", "Path to SVSM binary")
+	cmd.PersistentFlags().StringVar(&f.SvsmMeasurementPath, "svsm_measurement_path", "", "Path to SVSM measurement")
```

Comment:
```suggestion
	cmd.PersistentFlags().StringVar(&f.IGVMSNPMeasurementPath, "igvm_snp_measurement_path", "", "Path to SVSM measurement for SEV-SNP")
```

Reply:
Added "snp" but kept "svsm" instead of "igvm"